dockerfy/.env

```diff
@@ -7,3 +7,5 @@ MYSQL_PORT=33306
 MYSQL_USER=rvd_user
 MYSQL_PASSWORD=Pword12345*
 MYSQL_MY_DATABASE=ravada
+#modify TZ as required
+TZ=Europe/Athens
```

Comment:
Timezone goes here. Do not type anything else after timezone. I tried to add a comment after Europe/Athens and docker would not pass the variable to the containers

Reply:
We cannot set Athens by default.
Perhaps a good option is to include a .env_example where environment variables are self-documenting. And propose the user has to copy it to .env after apply your configuration.